images/linux/scripts/installers/edge.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -e
+################################################################################
+##  File:  edge.sh
+##  Desc:  Installs Edge
+################################################################################
+
+# Import Microsoft Edge GPG Key and Repository
+wget -O- https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > /usr/share/keyrings/microsoft-edge.gpg
+
+# Import the Microsoft Edge Repo
+echo 'deb [arch=amd64 signed-by=/usr/share/keyrings/microsoft-edge.gpg] https://packages.microsoft.com/repos/edge stable main' > /etc/apt/sources.list.d/microsoft-edge.list
+
+apt-get update
+
+apt-get install microsoft-edge-stable
+
+rm -f /etc/apt/sources.list.d/microsoft-edge.list
```

Comment:
if we remove the list then we need to add information about the repo to the readme. For example
https://github.com/actions/virtual-environments/blob/220c7ad69bfe75d87eb46d02278c0868ffc5f97a/images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1#L77-L81

Reply:
Ok, i'll do it